programs/create-genesis/main.cpp

```diff
@@ -173,13 +204,29 @@ int main(int argc, char** argv) {
             return 0;
         }
 
+        cr.read_options(vmap);
         cr.read_config(vmap);
-        cr.read_contracts();
 
-        genesis_create builder{};
-        builder.read_state(cr.info.state_file);
-        builder.write_genesis(cr.out_file, cr.ee_directory, cr.info, cr.genesis, cr.contracts);
+        state_object_visitor visitor;
+        state_reader reader{cr.info.state_file};
+        reader.read_state(visitor);
```

Comment:
please, move it back. this only required for create_genesis part, and create_genesis should be run every time (or else we can generate ee-genesis inconsistent with genesis.dat if change options between runs)